tests/swo_ccp_client/test_ccp_client.py

```diff
@@ -221,3 +222,17 @@ def test_request(mocker):
     mocker.patch("swo_ccp_client.client.CCPClient.get_ccp_access_token", return_value="auth-token")
     client.get_onboard_status("123")
     request_mock.assert_called_once()
+
+
+def test_keyvault_url_parsing(mocker, settings):
```

Comment:
Please refactor it to use parametrization here :-)

Reply:
❤️ Same thought! Sorry for the duplicated comment -> https://github.com/softwareone-platform/swo-aws-extension/pull/69/files#r2102221581.  I saw this one later